src/core/api/api.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import {
+  BASE_URL,
+  MOVIES_ENDPOINT,
+  SEARCH_OPTIONS,
+  SORT_OPTIONS,
+  DEFAULT_SORT_ORDER,
+  DEFAULT_SEARCH_LIMIT,
+  DEFAULT_TIME_OUT
+} from "@/core/api/constants";
+import { GENRE_OPTION, SIMILAR_MOVIES_LIMIT } from "@/core/constants";
+
+const ApiService = {
+  init() {
+    axios.defaults.baseURL = BASE_URL;
+    axios.defaults.timeout = DEFAULT_TIME_OUT;
+  },
+  getMovies() {
+    return axios.get(MOVIES_ENDPOINT).then(response => response.data);
+  },
+  getInitialMovies(
```

Comment:
When you have so many parameters, it could be more convenient to pass them as an object - that way you won't have to think about their ordering.

Reply:
Fixed. Thanks